http/client.go

```diff
@@ -434,3 +434,28 @@ func (c *Client) Events() events.Events {
 func (c *Client) MaxTxnRetries() int {
 	panic("client side database")
 }
+
+func (c *Client) CreateDocIndex(
+	ctx context.Context,
+	col client.Collection,
+	doc *client.Document,
+) error {
+	panic("client side database")
```

Comment:
todo: Please implement these :)

Reply:
We have plenty of http/cli funcs that receive docs as params

Reply:
Added error as discussed in syncup